sqlpage/templates/shell.handlebars

```diff
@@ -87,7 +87,7 @@
                     {{~#each (to_array menu_item)~}}
                         {{~#if (or (eq (typeof this) 'object') (and (eq (typeof this) 'string') (starts_with this '{')))}}
                             {{~#with (parse_json this)}}
-                                {{#if (or (gt (len this.title) 0) (gt (len this.icon) 0) (gt (len this.image) 0))}}
+                                {{#if (or (or (gt (len this.title) 0) (gt (len this.icon) 0)) (gt (len this.image) 0))}}
```

Comment:
wouldn't this work ?

```suggestion
                                {{#if (or (or this.title this.icon) this.image))}}
```

Reply:
I had some troubles filtering empty strings with plain checks. I double checked just now and it looks like it does work after all.